internal/testblobs/slow.go

```diff
@@ -71,18 +71,64 @@ func (slow *SlowBlobs) Open(ctx context.Context, ref storage.BlobRef) (storage.B
 	return slow.blobs.Open(ctx, ref)
 }
 
+// OpenSpecific opens a reader for the already-located blob, avoiding the potential need
+// to check multiple storage formats to find the blob.
+func (slow *SlowBlobs) OpenSpecific(ctx context.Context, ref storage.BlobRef, formatVer storage.FormatVersion) (storage.BlobReader, error) {
```

Comment:
This is not a blocking issue, but I would prefer a slightly more "specific" (ha...) name for these functions. Something like `OpenSpecificVer` would would imo. That way it is clear what we're being "specific" about, and how it differs from the regular `Open` call.